pkg/plugin/keyvault.go

```diff
@@ -102,29 +123,45 @@ func newKeyVaultClient(
 		keyVersion:       keyVersion,
 		vaultURL:         *vaultURL,
 		azureEnvironment: env,
+		keyID:            fmt.Sprintf("%x", sha256.Sum256([]byte(fmt.Sprintf("%s/keys/%s/%s", *vaultURL, keyName, keyVersion)))),
+		algorithm:        kv.RSA15,
```

Comment:
What exactly does this mean?  RSA isn't exactly the best crypto for disk encryption?

Reply:
I am not following this. We are using the same algo as before.